src/libs/Notification/PushNotification/index.js

```diff
@@ -2,6 +2,7 @@ import NotificationType from './NotificationType';
 
 // Push notifications are only supported on mobile, so we'll just noop here
 export default {
+    init: () => {},
```

Comment:
Now that this is actually called outside of the module, I think it makes sense to export it from the non-native file. But I'm not 100% sure it's necessary